setup.py

```diff
@@ -0,0 +1,33 @@
+import os
+from setuptools import find_packages, setup
+
+
+with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as readme:
+    README = readme.read()
+
+# Allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='archivematica-fpr-admin',
+    version='1.6.0',
+    packages=find_packages(exclude=['testproject']),
+    include_package_data=True,
+    license='AGPL',
+    author='Artefactual',
+    author_email='info@artefactual.com',
+    description='Format Policy Registry - Django app',
+    long_description=README,
+    url='https://github.com/artefactual/archivematica-fpr-admin',
+    classifiers=[
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'License :: OSI Approved :: GNU Affero General Public License v3',
+    ],
```

Comment:
Can you also include the Python language versions it supports? That's the most important bit of metadata IMO.  I think it's just `Programming Language :: Python :: 2.7` right now.